test/_log.js

```diff
@@ -1,9 +1,6 @@
 // Pull in the requirements.
```

Comment:
This provides inadequate coverage of the test. I would suggest bringing the testing log levels section back in and starting from there.